app/javascript/packages/secret-session-storage/index.spec.ts

```diff
@@ -138,4 +138,22 @@ describe('SecretSessionStorage', () => {
       expect(storage2.getItems()).to.deep.equal({});
     });
   });
+
+  describe('#clear', () => {
+    it('removes values from in-memory storage', async () => {
+      const storage = createStorage();
+      await storage.setItem('foo', 'bar');
+      storage.clear();
+
+      expect(storage.getItems()).to.deep.equal({});
+    });
+
+    it('removes values from persisted storage', async () => {
+      const storage = createStorage();
+      await storage.setItem('foo', 'bar');
```

Comment:
should this use STORAGE_KEY?
```suggestion
      await storage.setItem(STORAGE_KEY, 'bar');
```
otherwise it's not clear the test is checking the same thing

Reply:
While a bit confusing, there are two layers of keys here, since there's an object of values which are manipulated via `#setItem`, but the serialization of that object is persisted to a single value in `sessionStorage`. The `STORAGE_KEY` constant relates to the latter of these, and comes into play here for the initialization of the storage (through `createStorage`) and the verification of its destruction. Setting an item lets us make sure that nothing is left over after `clear`.

Reply:
ahhhh yes thanks for clarifying